gcp/api/server.py

```diff
@@ -131,7 +132,7 @@ def do_query(query, context, include_details=True):
       context.abort(grpc.StatusCode.INVALID_ARGUMENT, 'Invalid Package URL.')
       return None
 
-  to_response = lambda b: bug_to_response(b, include_details)
+  to_response = lambda b: bug_to_response(b, include_details)  # pylint: disable=unnecessary-lambda-assignment
```

Comment:
I just figured out what the lint here is trying to say. I think it wants you to do this:
```
...
      context.abort(grpc.StatusCode.INVALID_ARGUMENT, 'Invalid Package URL.')
      return None

      def to_response(b):
          return bug_to_response(b, include_details)
      
      next_page_token = None
...

```

This way it prevents people from accidentally modifying the `to_response` variable when it shouldn't be modified

Reply:
Thank you, that makes perfect sense now that you say it :-)

Reply:
Ideally I'd like to better understand why it's being done in the first place...

Reply:
It's done here because they wanted to add set the `include_details` parameter. If they just passed in bug_to_response, they will also need to add another argument to set whether to `include_details` or not